test/algoliax/utils_test.exs

```diff
@@ -4,18 +4,22 @@ defmodule Algoliax.UtilsTest do
   defmodule NoRepo do
     use Algoliax.Indexer,
       index_name: :algoliax_people,
-      attributes_for_faceting: ["age"],
-      searchable_attributes: ["full_name"],
-      custom_ranking: ["desc(updated_at)"],
+      algolia: [
+        attributes_for_faceting: ["age"],
+        searchable_attributes: ["full_name"],
+        custom_ranking: ["desc(updated_at)"]
+      ],
```

Comment:
(nit) I simply fixed the settings configuration, but it has 0 impact since those keys are not used by the functions tested in this module